srai/joiners/base.py

```diff
@@ -0,0 +1,31 @@
+"""Base class for joiners."""
+
+import abc
+
+import geopandas as gpd
+
+from ..regnionizers.base import BaseRegionizer
```

Comment:
fix the typo in the directory name and here
```suggestion
from ..regionizers.base import BaseRegionizer
```

Reply:
This time it was not me 😂 But will fix ;)